arrow/arrow_test.go

```diff
@@ -260,14 +272,22 @@ func TestArrowWriterHandlesNullValues(t *testing.T) {
 	}
 
 	readAndVerifyArrowFile(filePath, rows, false)
+
+	os.Remove(filePath)
 }
 
 func runConcurrentTest(compress bool) {
```

Comment:
Where is `compress` used? I deleted it and re-ran tests without changing anything.

Reply:
fixed